### PR TITLE
add volume internal block

### PR DESCRIPTION
**Pull request by losynix**
Volume internal block:

exec: "YABAR_VOLUME";
interval: 1;
internal-option1 : "default"; # device
internal-option2 : "Master 0"; # Mixer index (separated by a space)
internal-option3 : " "; # characters to display when sound is on or off (separated by a space)
internal-suffix: "%";

### CODE REVIEW

**Comment by losynix** [comment]
(the "libasound2-dev" package should be added in the travis config )

**Comment by losynix** [comment]
@jacksgt done ! thank you

**Comment by erebe** [comment]
Can we please integrate this pull request ?
I am using this patch for more than 1 month and everything is working fine.

**Comment by erebe** [comment]
Bump, any news ?

**Comment by NBonaparte** [comment]
Well, @geommer was last active in August, so it might take a while (if he ever even comes back).

**Comment by jacksgt** [comment]
That's funny, I just wrote George this monday :D 
A release 0.5 before Debian Stretch freeze would be great :-)

**Comment by erebe** [comment]
Happy new year bump :)

**Comment by NBonaparte** [comment]
@jacksgt has forked the project to https://github.com/yabar-dev/yabar, so development will continue there.

@losynix, could you make another PR on that repo?

**Comment by losynix** [comment]
@NBonaparte It will be done that by the end of the week !

**Comment by geommer** [comment]
I know it's too late, but I want to thank you @losynix for this good work, I am merging it but I have only one notice, I suggest we name the block "YABAR_VOLUME_ALSA" instead of just "YABAR_VOLUME" so we have a distinction if a "YABAR_VOLUME_PULSE" block is added in the future.

**Comment by jacksgt** [comment]
Hey @geommer , we discussed the VOLUME vs VOLUME_ALSA problematic over at [yabar-dev/yabar#2](https://github.com/yabar-dev/yabar/pull/2).

TL;DR This ALSA implementation also controls PulseAudio volume
